honeybee_radiance/cli/translate.py

```diff
@@ -149,6 +156,25 @@ def model_to_rad_folder(
     else:
         # re-serialize the Model and perform any checks
         model = Model.from_file(model_file)
+
+        if create_grids:
+            if not model.properties.radiance.has_sensor_grids:
+                sensor_grids = []
+                unit_conversion = {
+                    'Meters': 1,
+                    'Millimeters': 0.001,
+                    'Feet': 0.3048,
+                    'Inches': 0.0254,
+                    'Centimeters': 0.01
+                }
+                grid_size = 24 * 0.0254 / unit_conversion.get(model.units, 1)
+                offset = 30 * 0.0254 / unit_conversion.get(model.units, 1)
+                wall_offset = 12 * 0.0254 / unit_conversion.get(model.units, 1)
+                for room in model.rooms:
+                    sensor_grids.append(room.properties.radiance.generate_sensor_grid(
+                        grid_size, offset=offset, wall_offset=wall_offset))
```

Comment:
We should most likely introduce a property to let the user tag which rooms should have sensor grids. This will need a change in the schema but I'm just leaving a comment here for now.